source/documentation/appendix-docs/botor.md

```diff
@@ -80,6 +80,20 @@ reticulate::py_install('boto3')
 renv::install('botor')
 ```
 
+If this process goes wrong run
+
+```r
+renv::deactivate()
+```
+
+in the console and 
+
+```bash
+rm -rf renv renv.lock .Rprofile requirements.txt
```

Comment:
Why `requirements.txt` here? I don't believe we have any steps that create it

Reply:
`renv::snaphot()` will create it if python is used.

Reply:
Ah, I did not know that